### ISSUE TITLE
add some data about OCR confidence index

### ISSUE THREAD

**Issue by eroux** [OpenPecha/Toolkit#162]
It would be extremely helpful to have an average of the confidence index returned by the OCR in the opf. It could be at different levels:
- a global average per opf
- an average per volume
- an average per page, in a layer, althought that would make the files much bigger and is less urgent

**Comment by eroux** [comment]
perhaps median of the word-level (or page-level) confidence index is the best indicator... taking the line level will give an artificially lower value because of the noise I think

**Comment by kaldan007** [comment]
new meta:

id: I7E1A43F2
source: https://library.bdrc.io
source_file: null
initial_creation_type: ocr
imported: '2020-03-28T12:12:38+00:00'
last_modified: '2022-06-08T11:28:52.590761+00:00'
parser: https://github.com/OpenPecha-dev/openpecha-toolkit/blob/231bba39dd1ba393320de82d4d08a604aabe80fc/openpecha/formatters/google_orc.py
source_metadata:
  id: bdr:W3CN4314
  title: ཚད་མ་རིགས་པའི་གཏེར་གྱི་རྩ་བ།
  author: ''
  access: http://purl.bdrc.io/admindata/AccessOpen
  restrictedInChina: false
  opf_level_confidence: 0.9
  base:
    529C:
      image_group_id: I3CN8548
      title: ''
      total_pages: 62
      order: 1
      base_file: 529C.txt
      vol_level_confidence: 0.9

how does this look

**Comment by eroux** [comment]
thanks! a few remarks:

`opf_level_confidence` should be at the top level (not in `source_metadata`) and should be renamed to `ocr_word_median_confidence_index` (or `ocr_page_average_confidence_index` depending on the mode of calculation)

the `vol_level_confidence` should also be renamed in the same way

it's a bit dissatisfying to have information on the opf bases in `source_metadata` since (as its name indicates), this was originally meant to only record the BDRC data... why not something like:

```yml
id: I7E1A43F2
source: https://library.bdrc.io
source_file: null
initial_creation_type: ocr
imported: '2020-03-28T12:12:38+00:00'
last_modified: '2022-06-08T11:28:52.590761+00:00'
parser: https://github.com/OpenPecha-dev/openpecha-toolkit/blob/231bba39dd1ba393320de82d4d08a604aabe80fc/openpecha/formatters/google_orc.py
ocr_word_median_confidence_index: 0.9
source_metadata:
    id: bdr:W3CN4314
    title: ཚད་མ་རིགས་པའི་གཏེར་གྱི་རྩ་བ།
    author: ''
    access: http://purl.bdrc.io/admindata/AccessOpen
    restrictedInChina: false
  base:
    529C:
      source_metadata:
          image_group_id: I3CN8548
          title: ''
          total_pages: 62
      order: 1
      base_file: 529C.txt
      ocr_word_median_confidence_index: 0.9
```

(note also that the global value for the opf can't be computed by averaging the values of the volumes because they don't have the same number of words / pages)

**Comment by 10zinten** [comment]
@eroux that actually makes more sense, we will update the metadata model accrodingly.

**Comment by 10zinten** [comment]
here is the fix 125b0ab

**Comment by eroux** [comment]
Thanks @10zinten ! The issue was initially about OCR confidence index, so let's keep it open for that
